src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -254,10 +262,10 @@ export default withOnyx<IOURequestStepWaypointProps, IOURequestStepWaypointOnyxP
     recentWaypoints: {
         key: ONYXKEYS.NVP_RECENT_WAYPOINTS,
 
-        // Only grab the most recent 5 waypoints because that's all that is shown in the UI. This also puts them into the format of data
+        // Only grab the most recent waypoints because that's all that is shown in the UI. This also puts them into the format of data
```

Comment:
```suggestion
        // Only grab the most recent 20 waypoints because that's all that is shown in the UI. This also puts them into the format of data
```

Reply:
Bump on this comment too

Reply:
@eVoloshchak sorry I missed this comment 😅